shell/platform/darwin/macos/framework/Source/fixtures/flutter_desktop_test.dart

```diff
@@ -55,3 +57,9 @@ Picture _createSimplePicture() {
 void nativeCallback() {
   signalNativeTest();
 }
+
+@pragma('vm:entry-point')
+void backgroundTest() {
+  PlatformDispatcher.instance.views.first.render(SceneBuilder().build());
```

Comment:
@cbracken we decided the test should look like
```dart
Picture _createSimplePicture() {
  Paint paint = Paint();
  paint.color = Color(0xFFFF3A00);
  PictureRecorder baseRecorder = PictureRecorder();
  Canvas canvas = Canvas(baseRecorder);
  canvas.drawRect(Rect.fromLTRB(0.0, 0.0, 1000.0, 1000.0), paint);
  return baseRecorder.endRecording();
}

void main() async {
  window.render(SceneBuilder().build());
  // 1. should look black

  final SceneBuilder builder = SceneBuilder();
  builder.addPicture(Offset(1.0, 1.0), _createSimplePicture());
  builder.pushOffset(1.0, 2.0);
  builder.pop(); // offset
  window.render(builder.build());
  // 2. should have a red square

  window.render(SceneBuilder().build());
  // 3. should be black again.
}
```
but https://github.com/flutter/flutter/issues/113785 is preventing step 3 from producing the correct output. Before we can add a test like the one above, https://github.com/flutter/flutter/issues/113785 has to be fixed.